source/navitiacommon/navitiacommon/models.py

```diff
@@ -263,6 +265,45 @@ def __repr__(self):
         return '<Instance %r>' % self.name
 
 
+class TravelerProfile(db.Model):
+
+    coverage = db.Column(db.Text, db.ForeignKey('instance.name'), primary_key=True, nullable=True)
```

Comment:
it's possible to have a profile not associated to a a coverage? do we use it for define default profile?